bedrock/mozorg/templates/mozorg/advertising/solutions.html

```diff
@@ -0,0 +1,441 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "mozorg/advertising/base-advertising.html" %}
+
+{% block page_title %}Mozilla Ads - Solutions{% endblock %}
+
+{% block js %}
+  {{ super() }}
+  {{ js_bundle('protocol-modal') }}
+  {{ js_bundle('mozorg-advertising-solutions') }}
+{% endblock %}
+
+{% block content %}
+<main id="mza-main">
+  <!-- A page-level checkbox in the settings called "display page
+   components on dark background." toggles the existence of this div. -->
+  <div class="mza-c-main-dark-section">
+    <header class="mza-subpage-header">
+      <!-- the dark background toggle will flip on the `no-bottom-padding`
+       class here as well, since this h1 container should be in the template,
+       not the streamfield. -->
+      <div class="m24-c-content no-bottom-padding">
+        <h1>Solutions</h1>
+      </div>
+    </header>
+
+    <!-- "Subheading" Component. As a streamfield block, this is always an h2. -->
+    <div class="mza-subpage-header">
+      <div class="m24-c-content no-bottom-padding">
+        <hr/>
+        <h2 class="mza-section-heading">Firefox</h2>
+        <p class="mza-highlight-text">Firefox stands as the most trusted alternative to one-size-fits-all, privacy-invading default browsers. Driven by Mozilla’s mission to ensure the internet is open and accessible to everyone, Firefox puts users in control.</p>
+      </div>
+    </div>
+
+    <!-- existing figure plus stat block from homepage -->
+    <div class="m24-c-content padding-top-0">
+      <div class="mza-figure-grid reversed-on-large">
+        <aside class="mza-stat">
+          <span class="mza-stat-text">
+            <span class="mza-stat-number is-smaller">210M+</span>
+            <span class="mza-stat-label">monthly active Firefox users</span>
+          </span>
+        </aside>
+        <figure class="mza-figure">
+          {{ resp_img(
+            url="img/mozorg/advertising/solutions/ads-solutions-page-1.png",
+            optional_attributes={
+              'loading': 'lazy',
+              'alt': 'stylized screenshot of Firefox browser'
```

Comment:
```suggestion
              'alt': 'Stylized screenshot of Firefox browser.'
```

Yay, alt text!

Including a period at the end of alt gives the listener an audio cue that the text has finished (IMHO screen readers should do this automatically, but here we are).

Screen readers *do* include an audio cue that it's an image so it's time saving to not repeat image/photo/screenshot etc. as part of the alt.